tests/ble_driver_test.py

```diff
@@ -4,70 +4,55 @@
 config.__conn_ic_id__ = 'NRF52'
 from pc_ble_driver_py.ble_driver import Flasher
 
+class FlasherParserTestCase(unittest.TestCase):
+ 
+    def test_valid_parse(self):
```

Comment:
Could add an invalid test case as well, test_invalid_parse, to validate the error path.